### DISCUSSION TITLE
[BUG] Blade of Eternal Darkness

### DISCUSSION THREAD

**Discussion by Sealfate** [chromiecraft/chromiecraft#2416] [General]
Supposedly proc should be able to crit but does not work on this server.

Proc damage is flat 100. It should be able to crit for 200 damage. It should not be affected by crit amplifier talents though.

Critical chance should be determined by how much spell critical chance one character has.

Mana return is working properly. Does not proc of DoTs which works correctly. Only of direct damage and this is also good.

Out of 220+ proc from couple of days ago in dungeon i did not have a single crit.

I wish this can be fixed so many people can enjoy this sweat and rare epic weapon while it last. Once we get into Molten Core there will be much stronger options and upgrades and sadly this beautiful dagger will most likely collect dust in bank.

**Comment by Yehonal** [comment]
This is not the place where to report a bug, our triage team does not read the forum.

Please, use the correct guidelines: https://github.com/chromiecraft/chromiecraft#readme
